packages/strapi-plugin-upload/package.json

```diff
@@ -31,7 +31,7 @@
     "react-copy-to-clipboard": "^5.0.3",
     "react-dom": "^16.9.0",
     "react-intl": "4.5.0",
-    "react-redux": "^7.2.3",
+    "react-redux": "^7.2.2",
```

Comment:
you should remove the ^ in the deps to avoid the upgrade if it breaks some stuff